libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -1492,5 +1492,13 @@ type SystemSettingsAPI =
         :> "system"
         :> "settings"
         :> "unauthorized"
-        :> Get '[JSON] SystemSettings
+        :> Get '[JSON] SystemSettingsPublic
     )
+    :<|> Named
+           "get-system-settings-authorized"
```

Comment:
I would call this endpoint `get-system-settings` and the unauthorized one `get-system-settings-unauthorized`, because that reflects the path. However, this is likely a matter of taste.